dev_guide/integrating_external_services.adoc

```diff
@@ -53,12 +53,20 @@ ignore the service and allow you to specify endpoints manually:
 ----
 {
   "kind": "Service",
-  "apiVersion": "v1beta3",
+  "apiVersion": "v1",
   "metadata": {
     "name": "example-mysql-service"
   },
-  "spec": { <1>
```

Comment:
Yeah, I'm not an expert in that area, but @mfojtik was dealing with them, so I'm hoping to get his answer on this. Clearly that was an error in the docs.